src/Fhir.Anonymizer.Shared.FunctionalTests/Configurations/substitute-conflict-rules.json

```diff
@@ -0,0 +1,22 @@
+﻿{
+  "fhirPathRules": [
```

Comment:
Maybe we should add a test for multiple substitute rule?

Reply:
Thanks. Added.